<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: white;
    border-radius: 20px;
}

QPushButton
{
    color: white;
    background-color: #6357B7;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
}

QPushButton:Pressed
{
    background-color: #C397E4;
}

QPushButton:disabled
{
    color: #6357B7;
    background-color: white;
    border-radius: 0px;
}

QLineEdit
{
    color: #6357B7;
    background-color: white;
    border : 2px solid #6357B7;
    border-radius: 15px;
    font: 10pt &quot;Bahnschrift&quot;;
    border: 1px solid #6357B7;
    padding: 10px;
}

QLabel
{
    background-color: transparent;
    color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}

QComboBox
{
    border: 1px solid #6357B7;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
	color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}

QTextEdit
{
    color: #6357B7;
    background-color: white;
    border : 2px solid #6357B7;
    border-radius: 15px;
    font: 14pt &quot;Bahnschrift&quot;;
    border: 1px solid #6357B7;
    padding: 13px;
}

QTableWidget
{
	color: #6357B7;
    font: 12pt &quot;Bahnschrift&quot;;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Reporte de Aerolineas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Aerolineas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Generar / Actualizar Reporte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Id Aerolinea</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Aerolinea</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Reporte de Estadisticas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>167</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Generar / Actualizar Reporte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Aerolinea</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cantidad / Valor</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
